modules/javafx.controls/src/test/java/test/javafx/scene/control/TestListenerHelper.java

```diff
@@ -280,7 +280,7 @@ public void testInvalidationListener_Callback() {
         SimpleStringProperty p = new SimpleStringProperty();
         AtomicInteger ct = new AtomicInteger();
 
-        h.addInvalidationListener(p, (cur) -> ct.incrementAndGet());
+        h.addInvalidationListener(() -> ct.incrementAndGet(), p);
```

Comment:
Now this is somewhat redundant with `testInvalidationListener_MultipleProperties`, since it tests the same flavor of method -- unlike `addChangeListener`, there are only two patterns, not three. Although showing that it works with a single property is possibly useful (the same could be said for the ChangeListener tests), so I don't mind if you prefer to keep it.